### PR TITLE
Add com.szibele.e-juice-calc

### PR DESCRIPTION
**Pull request by rszibele**
Hi,

This adds e-juice-calc which is available under the GPLv2 to Flathub: https://gitlab.com/rszibele/e-juice-calc

The Haskell dependencies within the manifest have been auto-generated by https://gitlab.com/rszibele/stackpak .

Thanks!

### CODE REVIEW

**Comment by nedrichards** [comment]
bot, build com.szibele.e-juice-calc

**Comment by rszibele** [comment]
Is it possible for me to manually start the build as well?

**Comment by rszibele** [comment]
bot, build com.szibele.e-juice-calc

**Comment by nedrichards** [comment]
bot, build com.szibele.e-juice-calc

**Comment by rszibele** [comment]
Awesome, it built for all platforms (i386, x86_64, arm, aarch64) without issues.
https://flathub.org/builds/#/builders/2/builds/4071

File: com.szibele.e-juice-calc.json [hunk 1]
```diff
@@ -0,0 +1,1173 @@
...
+        "rm -rf /app/lib/armv7-linux-ghc-8.2.2",
+        "rm -rf /app/share/doc/armv7-linux-ghc-8.2.2",
+        "rm -rf /app/share/armv7-linux-ghc-8.2.2",
+        "rm -rf /app/lib/aarch64-linux-ghc-8.2.2",
+        "rm -rf /app/share/doc/aarch64-linux-ghc-8.2.2",
+        "rm -rf /app/share/aarch64-linux-ghc-8.2.2"
+    ],
+    "command": "e-juice-calc",
+    "finish-args": [
+        "--socket=x11"
```
Comment (nedrichards) [inline com.szibele.e-juice-calc.json]: this needs --share=ipc as well.

**Comment by nedrichards** [comment]
Other than that nit, it looks fine. I'm making a *lot* of style allowances for the fact that this is autogenerated.

**Comment by nedrichards** [comment]
The alphabetical order is the biggest issue. @TingPing is the best stylist amongst the review team and I suspect would be best to help here.

**Comment by nedrichards** [comment]
Oh, a quick look upstream shows me that the appdata doesn't have OARS: https://hughsie.github.io/oars/generate.html

**Comment by rszibele** [comment]
For reference, the appdata can be found here: https://gitlab.com/rszibele/e-juice-calc/blob/1.0.5/packaging/com.szibele.e-juice-calc.appdata.xml

**Comment by nedrichards** [comment]
bot, build com.szibele.e-juice-calc

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/com.szibele.e-juice-calc

**Pull request closed without merging**